frog/imports/api/objects.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+// $FlowFixMe
+import { Mongo } from 'meteor/mongo';
+import type { ObjectT } from 'frog-utils'; // eslint-disable-line
```

Comment:
Because I import twice from 'frog-utils' which makes eslint complain. I didn't find quickly how to import both types and normal imports from one import statement.